app/controllers/test_controller.rb

```diff
@@ -225,6 +225,39 @@ def suggest_similarity_item
     render_success 'suggest_similarity', pm1
   end
 
+  def create_imported_standalone_fact_check
+    team = Team.current = Team.find(params[:team_id])
+    user = User.where(email: params[:email]).last
+    description = params[:description]
+    context = params[:context]
+    title = params[:title]
+    summary = params[:summary]
+    url = params[:url]
+    language = params[:language] || 'en'
+
+    # Create ClaimDescription
+    claim_description = ClaimDescription.create!(
+      description: description,
+      context: context,
+      user: user,
+      team: team
+    )
+
+   # Set up FactCheck
+  fact_check = FactCheck.new(
+    claim_description: claim_description,
+    title: title,
+    summary: summary,
+    url: url,
+    language: language,
+    user: user,
+    publish_report: true,
+    report_status: 'published'
+  )
+  fact_check.save!
+  render_success 'fact_check', fact_check
```

Comment:
Please fix the indentation on this block.

Reply:
:white_check_mark: